canonical-websites/maas.io
# Fix the error pages, fix the app

Since Django 2 upgrade, 404 pages were broken. This led to /_status/check also being broken, which made the production rollouts fail.

This fixes that.

QA
--

`./run --env DJANGO_DEBUG=false`

Check the site. Go to http://0.0.0.0:8006/_status/check, check you see a git commit ID (with a 200 status)
Go to http://0.0.0.0:8006/some/error/page, check you see a 404 page.
